recipes/ffnet/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```

Comment:
I think it is reasonable to remove both the `bld.bat` and `build.sh` in favor of just using the `script` entry in the `meta.yaml`:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26